gpflow/kernels/misc.py

```diff
@@ -101,8 +101,9 @@ def _J(self, theta):
             return 3.0 * tf.sin(theta) * tf.cos(theta) + (np.pi - theta) * (
                 1.0 + 2.0 * tf.cos(theta) ** 2
             )
+        assert False, f"Don't know how to handle order {self.order}."
```

Comment:
(if you want codecov to leave you alone, you could instead replace the `elif self.order == 2:` above with an `else:` followed by an `assert self.order == 2`)

Reply:
> (if you want codecov to leave you alone, you could instead replace the `elif self.order == 2:` above with an `else:` followed by an `assert self.order == 2`)

Good idea. Done.